tests/lmeval/configs/vl_fp8_dynamic_per_token.yaml

```
@@ -0,0 +1,18 @@
+cadence: weekly
+model: Qwen/Qwen2.5-VL-7B-Instruct
+model_class: TraceableQwen2_5_VLForConditionalGeneration
+scheme: FP8_DYNAMIC
+lmeval:
+  model: "hf-multimodal"
+  model_args:
+    dtype: bfloat16
+    add_bos_token: True
+    convert_img_format: True
+  task: mmmu_val_literature
+  num_fewshot: 0
+  use_stderr_atol: True
+  batch_size: 8
+  # dense model achieves accuracy of 0.9 +/ 0.0557
+  metrics:
+    acc,none: 0.8667
+    acc_stderr,none: 0.0557
```

This might be easier to parse
```suggestion
    acc,none: (0.8667, 0.0557)
```

```python3
value, std = value if isinstance(value, tuple) else value, 0.05
```